protoutil/blockutils_test.go

```diff
@@ -606,11 +610,15 @@ func TestVerifyTransactionsAreWellFormed(t *testing.T) {
 		},
 	} {
 		t.Run(tst.name, func(t *testing.T) {
-			err := protoutil.VerifyTransactionsAreWellFormed(tst.block)
-			if tst.expectedError == "" {
-				require.NoError(t, err)
+			if tst.block == nil || tst.block.Data == nil {
+				require.Error(t, errors.New("empty block"))
```

Comment:
what does this check? that an error is an error ;-)

Reply:
oh my god